src/main/java/org/example/homework_nr_8/Animal.java

```diff
@@ -0,0 +1,13 @@
+package org.example.homework_nr_8;
+
+public interface Animal {
+    void eat();
+    void run();
+    default void drink(){
```

Comment:
add a simple implementation here. If you want it to do nothing, make it print out to console that it's not overriden.